README.md

```diff
@@ -1,32 +1,52 @@
 # ChromeIDE
 Parallax microcontroller development environment based on Chrome application technologies.
 
-## Installing from Source
+## Build from Source
 
-1. Clone this respository.  Choose your file destination from Terminal and
+This is a one-time procedure needed to initially run the application.  To build the application will need the Node javascript library.  If you don't have it, then:
 
-	``` git clone git@github.com:parallaxinc/ChromeIDE ```
+1. Install Node by going to [nodejs.org/download](https://nodejs.org/download/) and selecting the option that is right for your system.
 
-	![clone repo](readme-images/clone-chromeIDE.png)
+1. Clone this respository.  Assuming you have `git` installed (if not, [go here](http://git-scm.com/book/en/v2/Getting-Started-Installing-Git)), choose your file destination from the command-line (CLI like Terminal), and
 
+  ``` git clone git@github.com:parallaxinc/ChromeIDE ```
 
-2. Open your Chrome browser and navigate to
+  ![clone repo](readme-images/clone-chromeIDE.png)
+
+1. Go to the project root directory with
+
+  ``` cd  ChromeIDE/ ```
+
+1. Get all the project dependencies
+
+  ``` npm install ```
+
+1. Bundle and build the application by entering
+
+  ```npm run build```
```

Comment:
add a note about `npm run build -- --watch` to to build on any change during development